clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -269,6 +286,142 @@
 // CHECK-M-EXT: __riscv_mul 1
 // CHECK-M-EXT: __riscv_muldiv 1
 
+// RUN: %clang --target=riscv32-unknown-linux-gnu \
```

Comment:
(For newer tests, ensure the continuation line is indented by 2 spaces)

Reply:
Should I just reformat this file?